sanitizer/_text_pelias_parser.js

```diff
@@ -95,6 +95,11 @@ function parse (clean) {
   // '    VVVV NN SSSSSSS AAAAAA PPPPP      '
   let mask = solution.mask(t);
 
+  // log information about the selected solution
+  logger.info('pelias_parser_solution', {
+    score: solution.score.toFixed(2)
```

Comment:
It would be interesting to have the input too here

Reply:
The input is already being logged at https://github.com/pelias/api/blob/ef2969b0415a4d182247baa5e5624be73ac32186/sanitizer/_text_pelias_parser.js#L66

I decided to split up those log sections so the `pelias_parser` log entry provides an overview of the parse and then we can have one or more `pelias_parser_solution` lines which log info about specific solutions.

At this stage, we only pick the first solution, but we could explain this later and if we end up having multiple `pelias_parser_solution` log lines it'll be cleaner to have the input only logged once since it doesn't change.

Reply:
This was to have a simple way to analyze the logs.
For example if we want to quickly know which queries have a low score and the associated input all at once (using kibana or whatever). Unless it is possible to have links between log lines ? :thinking: